src/components/Day.vue

```diff
@@ -61,7 +61,8 @@ export default {
       type: String
     },
     currentDateStyle:{
-      default:null,
+      default:'border: 1px solid #00c690',
```

Comment:
No need to define the default again here, we can just mark this prop as required :)

Reply:
got you i'll update it now. Thanks for the point